apps/rehabstod/src/components/Form/FormattedNumberInput.tsx

```diff
@@ -26,6 +28,9 @@ export function FormattedNumberInput({
   }
 
   function maxLengthReached(event: React.KeyboardEvent<HTMLInputElement>) {
+    if (ignoreInputLimit) {
+      return false
+    }
     return event.currentTarget.value.length === 2 && event.key !== 'Backspace' && selection?.type !== 'Range'
```

Comment:
Could we instead get the length from the max attribute?

`event.currentTarget.value.length === `${max}`.length`

Reply:
Oh, thats brilliant! Great suggestion, updated the PR!